src/pages/DemoForGitRepository.java

```diff
@@ -10,6 +10,6 @@ public void demoGit()
 		System.out.println("second change for pull command in git");
         System.out.println("New year gonna be awesome dont worry");
         System.out.println("going to integrate github in jenkins tool");
-        System.out.println("working on git pull request in compny");
+        System.out.println("working on git pull request in compny now you will see the changes");
```

Comment:
Great work but i want you to add one more line to it so it will look properly without any mistake.